folium/utilities.py

```diff
@@ -305,7 +305,11 @@ def base(x):
     quants = [0, 50, 75, 85, 90]
```

Comment:
@koldunovn Can you get this line down (in between 308-309), so we can easily see that the two options are doing.